.github/release-drafter.yml

```diff
@@ -0,0 +1,45 @@
+name-template: 'v$RESOLVED_VERSION'
+tag-template: 'v$RESOLVED_VERSION'
+
+template: |
+  ## Changes
+  $CHANGES
+  ### Contributors
+  $CONTRIBUTORS
+categories:
+  - title: '🚀 New Features'
+    labels:
+      - 'type: feature'
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'type: bugfix'
+  - title: '📝 Documentation'
+    labels: 'type: docs'
+  - title: '🔐 Security'
+    labels:
+      - 'type: security'
+  - title: '⚙️ Internal Changes'
+    labels:
+      - 'type: chore'
+      - 'type: ci'
+      - 'type: perf'
+      - 'type: refactor'
+      - 'type: test'
+change-template: '- $TITLE (#$NUMBER • @$AUTHOR)'
+change-title-escapes: '\<*_&' # You can add # and @ to disable mentions, and add ` to disable code blocks.
```

Comment:
You will have to add the `tag-prefix: v` option
As your tags are currently all using that prefix.